### DISCUSSION TITLE
`gatsby-remark-images` only allows loading: lazy

### DISCUSSION THREAD

**Discussion by MelleNi** [gatsbyjs/gatsby#27888] [Help] [answered]
## Description

When attempting to run a gatsby develop environment with `gatsby-remark-images` with the option `loading` set to anything but `lazy` results in `ReferenceError: eager is not defined` or `ReferenceError: auto is not defined`.

### Steps to reproduce

Include the option `loading: auto,` or `loading: eager,` in `gatsby-remark-images`

### Expected result

It should start the develop environment without errors and enable either of those options for the plugin.

### Actual result

An error breaks the process

### Environment

  System:
    OS: macOS 10.15.6
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 12.19.0 - ~/.nvm/versions/node/v12.19.0/bin/node
    npm: 6.14.8 - ~/.nvm/versions/node/v12.19.0/bin/npm
  Languages:
    Python: 3.8.5 - /Users/melle/.pyenv/shims/python
  Browsers:
    Chrome: 86.0.4240.183
    Edge: 86.0.622.43
    Firefox: 82.0.2
    Safari: 13.1.2
  npmPackages:
    gatsby: ^2.24.85 => 2.24.85 
    gatsby-image: ^2.4.21 => 2.4.21 
    gatsby-plugin-feed: ^2.5.19 => 2.5.19 
    gatsby-plugin-google-analytics: ^2.3.19 => 2.3.19 
    gatsby-plugin-manifest: ^2.4.35 => 2.4.35 
    gatsby-plugin-offline: ^3.2.37 => 3.2.37 
    gatsby-plugin-percy: ^0.1.4 => 0.1.4 
    gatsby-plugin-purgecss: ^5.0.0 => 5.0.0 
    gatsby-plugin-react-helmet: ^3.3.14 => 3.3.14 
    gatsby-plugin-sharp: ^2.6.43 => 2.6.43 
    gatsby-remark-copy-linked-files: ^2.3.19 => 2.3.19 
    gatsby-remark-images: ^3.4.1 => 3.4.1 
    gatsby-remark-prismjs: ^3.5.16 => 3.5.16 
    gatsby-remark-responsive-iframe: ^2.4.17 => 2.4.17 
    gatsby-remark-smartypants: ^2.3.13 => 2.3.13 
    gatsby-source-filesystem: ^2.3.35 => 2.3.35 
    gatsby-transformer-remark: ^2.8.46 => 2.8.46 
    gatsby-transformer-sharp: ^2.5.19 => 2.5.19 
  npmGlobalPackages:
    gatsby-cli: 2.12.114

**Comment by sidharthachatterjee** [comment, marked as answer]
Based on the error, it appears you're trying to set loading to a variable name? That ought to be `loading: "eager"`

**Comment by MelleNi** [comment]
> Based on the error, it appears you're trying to set loading to a variable name? That ought to be `loading: "eager"`

Makes sense, wasn't clear from the documentation. Any idea why `lazy` without quotes does not return the same error, in that case?
